scilpy/io/utils.py

```diff
@@ -463,6 +464,35 @@ def verify_compatibility_with_reference_sft(ref_sft, files_to_verify,
                              .format(file))
 
 
+def verify_compatibility(parser, list_files):
```

Comment:
I think this title is a bit too similar to "is_header_compatible", it feels like it will do the same when looking only at the name.
verify_compatibility_multiple_files? I don't know.

Reply:
Also, this loop is probably done in many many other scripts! Could be re-used!

Reply:
I think I got everything cover now. I also added the pair comparison instead of using only one reference.